themes/default/templates/package.mtt

```diff
@@ -1,18 +1,7 @@
 ::use 'main.mtt'::
 
 ::if full == ""::
-	<h1>Haxe API documentation <small ::cond api.isDefined("version")::>version ::api.getValue('version')::</small></h1>
-	<p>Haxe is an open source toolkit based on a modern, high level, strictly typed programming language, a cross-compiler, a complete cross-platform standard library and ways to access each platform's native capabilities.</p> 
-	<h3>Getting Started With Haxe</h3>
-	<ul>
-		<li>Take a look at our <a href="http://haxe.org/documentation/introduction/">introduction</a></li>
-		<li>Read through the <a href="http://haxe.org/manual/">Haxe Manual</a></li>
-		<li>Look at these <a href="http://haxe.org/use-cases/">use cases for Haxe</a></li>
-		<li>Find and install <a href="http://lib.haxe.org/t/all/">popular Haxe libraries</a></li>
-		<li>Learn by example with the <a href="http://code.haxe.org">Haxe Code Cookbook</a></li>
-	</ul>
-	<hr/>
-	<h3>Top Level</h3>
+	::use "start.mtt"::::end::
 ::elseif full.split(".").length==1::
```

Comment:
I think this also needs to be split, or maybe make it optional using a flag, since it's very specific for the Haxe API.